src/lib/components/forms/form/form-components/multi-select/multi-select.component.html

```diff
@@ -0,0 +1,17 @@
+<mat-form-field class="w-100" [tcTooltip]="tooltipText" [placement]="tooltipPlacement">
+    <mat-label>
+        {{ name }}
+    </mat-label>
+    <mat-select
+        multiple
+        [(ngModel)]="value"
+        (ngModelChange)="_onChangeCallback($event)"
+        (blur)="_onTouchedCallback()"
+        [required]="isRequired"
+        [disabled]="isDisabled"
+    >
+        <mat-option *ngFor="let item of listBoxValues | selectSort: 'asc':'position'" [value]="item.value">
```

Comment:
Die Liste sollte hier nicht sortiert werden. Die Reihenfolge der options ist die Basis für die Sortierung der Liste der selektierten Werte. Daher sollten wir da nicht eingreifen.
```suggestion
        <mat-option *ngFor="let item of listBoxValues" [value]="item.value">
```